### PR TITLE
Change docs according self-signed certificate flag automation

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### What does this PR do?
Updates docs according to automation of `--self-signed-certificate` flag.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16764
https://github.com/eclipse/che/issues/15301

### Specify the version of the product this PR applies to. 
Eclipse Che 7.15.0

### CODE REVIEW

File: src/main/pages/che-7/installation-guide/assembly_installing-che-in-tls-mode-with-self-signed-certificates.adoc [hunk 1]
```diff
@@ -16,21 +16,55 @@ summary:
 
 :context: installing-{prod-id-short}-in-tls-mode-with-self-signed-certificates
 
-WARNING: This guide is for development only. Do not use self-signed certificates in production.
+WARNING: Self-signed certificates is usually used in development and evaluation enviromnents. If you want to use them in production you should understand the consequences.
```
Comment (rkratky) [inline src/main/pages/che-7/installation-guide/assembly_installing-che-in-tls-mode-with-self-signed-certificates.adoc]:
```suggestion
WARNING: Self-signed certificates are usually used in development and evaluation environments. Use in production environments is not recommended.
```

Comment (AndrienkoAleksandr) [inline src/main/pages/che-7/installation-guide/assembly_installing-che-in-tls-mode-with-self-signed-certificates.adoc]: enviromnents -> environments

**Review by rkratky — CHANGES_REQUESTED** [review]
Submitted my review and suggested changes in #1340.

**Review by rkratky — APPROVED** [review]

**Comment by rkratky** [comment]
@mmorhun, thanks for the updates. As mentioned in #1340, I'm merging this in your absence, but I'll be happy to revisit it if needed.

**Pull request merged**